### PR TITLE
Settings: Add Show Network Info modal

### PR DESCRIPTION
**Pull request by niteshbalusu11**
Added network info card. 

https://github.com/hsjoberg/blixt-wallet/issues/1064

### CODE REVIEW

File: src/state/Lightning.ts [hunk 1]
```diff
@@ -50,6 +52,7 @@ export interface ILightningModel {
 
   setBestBlockheight: Action<ILightningModel, number>;
 
+  networkInfo: lnrpc.NetworkInfo;
```
Comment (hsjoberg) [inline src/state/Lightning.ts:55]: nit: this either needs to be `networkInfo?: lnrpc.NetworkInfo;` or you have to initialize dummy data.

Seeing how `src/windows/Settings/LightningNetworkInfo.tsx` expects `networkInfo` to possibly be `null`, the former is reasonable  and works like `nodeInfo`.

File: src/windows/Settings/LightningNetworkInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+import { formatISO, toast } from "../../utils";
+import { useStoreState, useStoreActions } from "../../state/store";
+
+import { useTranslation,TFunction } from "react-i18next";
+import { namespaces } from "../../i18n/i18n.constants";
+import Long from "long";
+
+let t:TFunction;
+
+//const { t, i18n } = useTranslation(namespaces.settings.lightningNodeInfo)
```
Comment (hsjoberg) [inline src/windows/Settings/LightningNetworkInfo.tsx:17]: Please clear this stuff

File: src/windows/Settings/LightningNetworkInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    >
+      <Text style={{ fontWeight: "bold" }}>{title}:{"\n"}</Text>
+      {Array.isArray(data) && data.join("\n")}
+      {!Array.isArray(data) && data}
+    </Text>
+  );
+};
+
+export default function LightningNetworkInfo() {
+  t = useTranslation(namespaces.settings.lightningNetworkInfo).t;
```
Comment (hsjoberg) [inline src/windows/Settings/LightningNetworkInfo.tsx:42]: Do `const { t } = useTranslation(namespaces.settings.lightningNodeInfo)` instead. Also needs to be in the `MetaData` function above

**Review by hsjoberg — CHANGES_REQUESTED** [review]
LGTM after some nits 🔥

**Comment by niteshbalusu11** [comment]
> LGTM after some nits 🔥

Fixed.

**Comment by hsjoberg** [comment]
Nice stuff 👍

**Pull request merged**